tbx/project_styleguide/templates/patterns/pages/team/team_listing.html

```diff
@@ -8,7 +8,12 @@
         <ul class="team-listing grid__team-listing">
             {% for member in people %}
                 <li class="team-listing__item">
-                    {% srcset_image member.image format-webp fill-{230x230,370x370} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" %}
+                    {# lazy load after the first five #}
+                    {% if forloop.counter > 5 %}
+                        {% srcset_image member.image format-webp fill-{230x230,370x370} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" loading="lazy" %}
+                    {% else %}
+                        {% srcset_image member.image format-webp fill-{230x230,370x370} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" %}
+                    {% endif %}
```

Comment:
I wish we could do

```django
                        {% srcset_image member.image format-webp fill-{230x230,370x370} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" {% if forloop.counter > 5 %}loading="lazy"{% endif %} %}
```

Which would avoid repeating the code. Unfortunately that doesn't work :disappointed:: 

<details><summary>Details</summary>
![image](https://github.com/torchbox/torchbox.com/assets/7713776/4b6f4933-400a-4234-bb7e-aa4495b7202e)
</details>